docs/UserGuide.md

```diff
@@ -262,6 +263,25 @@ Calculates and displays the total revenue based on all the completed events.
 
 **Format**: `revenue`
 
+### View Upcoming Events: `remind` <a id = "view-upcoming-events"></a>
+
+Displays events that start within a specified number of days.
+
+**Format**: `remind DAYS`
+
+- Displays events that start within the specified number of `DAYS`.
+  - `DAYS` must be a positive integer 1, 2, 3, …
+
+**Example**:
+
+Assume the current time is 22-03-2023 11:00.
+
+- `remind 2`
+  - Events that start within 2 days will be displayed. These are events that start on:
+    * 22-03-2023 after 11:00
+    * 23-03-2023
+    * 24-03-2023
```

Comment:
Is the timing a strict 24 hours based per day or will events on 24 after 11:00 be displayed if the specified `DAYS` is 2?

Good to clarify!

Reply:
Will do 👍